src/pal/src/arch/i386/dispatchexceptionwrapper.S

```diff
@@ -17,9 +17,9 @@
 #include "unixasmmacros.inc"
 
 #if defined(__x86_64__)
-#define PAL_DISPATCHEXCEPTION __Z21PAL_DispatchExceptionmmmmmmP8_CONTEXTP17_EXCEPTION_RECORDP11MachMessage
+#define PAL_DISPATCHEXCEPTION __Z21PAL_DispatchExceptionmmmmmmP8_CONTEXTP17_EXCEPTION_RECORDP17MachExceptionInfo
```

Comment:
I was thinking about this in the past already, but since you are touching the name, you might make the PAL_DispatchException `extern "C"` and then use C_FUNC(PAL_DispatchException) in the asm code below instead of this mangled name.